src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java

```diff
@@ -13,6 +13,12 @@ final class InputColumnLimit {
         // violation above 'Line is longer than 100 characters (found 105).'
       };
 
+  @SuppressWarnings({"LineLength"})
+  int aaaarealllllllllllllllllyyyyyyyyyyylllllllloooooooooooooooonnnnnnnnnnnnnnnnnggggggggggvvvvvvaarriaaabllee = 99; // ok, suppression used, check name in camelcase
+
+  @SuppressWarnings({"linelength"})
+  int aaaarealllllllllllllllllyyyyyyyyyyylllllllloooooooooooooooonnnnnnnnnnnnnnnnnggggggggggvvvvvvaarriaaabllee2 = 99; // ok, suppression used, check name in lowercase
```

Comment:
Please add examples on other identifiers, classes, methods , cast to types, exception catching, just type long usage with not long name.

What I got a worry is that annotation can not be placed at all places where identifiers can used.

Reply:
If we find such code, we will just offer one more way of suppression.

Reply:
done

https://github.com/checkstyle/checkstyle/blob/3adc3090e37e6fd20d1f285b612b0361ad3cef43/src/it/resources/com/google/checkstyle/test/chapter4formatting/rule44columnlimit/InputColumnLimit.java#L55-L103

> What I got a worry is that annotation can not be placed at all places where identifiers can used.

If there's a block which contains such block then we can place the annotation at the beggining of that block, this way the violation will get suppressed but down side is that the suppression will apply to whole block as explained here: https://github.com/checkstyle/checkstyle/pull/15460#discussion_r1707521991